### PR TITLE
[MAIN] [STRATCONN] 4119 klaviyo fixed blank audience creation.

### PR DESCRIPTION
**Pull request by AnkitSegment**
In this PR resolved the issue creating blank audience (List Id) on klaviyo, 
Now user can add list_id on audience settings page and be able to add audience on existing audience. 

JIRA TICKET: https://segment.atlassian.net/browse/STRATCONN-4119

Screenshot 2024-09-18 at 7 46 21 PM

\
\
Here, I created a new audience and added list_id on audienceField, Now we can verify that the new list name "Segment Test" is not created in Klaviyo and users are getting added to the provided list_id. 

\
\

Screenshot 2024-09-19 at 1 03 24 PM
Screenshot 2024-09-19 at 1 03 16 PM
Screenshot 2024-09-19 at 12 57 38 PM

_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [X] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [X] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [X] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by itsarijitray** [comment]
LGTM!

**Review by itsarijitray — APPROVED** [review]

File: packages/destination-actions/src/destinations/klaviyo/generated-types.ts [hunk 1]
```diff
@@ -6,3 +6,12 @@ export interface Settings {
...
   api_key: string
 }
+// Generated file. DO NOT MODIFY IT BY HAND.
+
+export interface AudienceSettings {
+  /**
+   * Insert the ID of the default list that you'd like to subscribe users to when you call .identify().
+   *        NOTE: List ID takes precedence set within Actions.
+   */
+  listId?: string
```
Comment (Innovative-GauravKochar) [inline packages/destination-actions/src/destinations/klaviyo/generated-types.ts:16]: I wonder If we can make this dropdown and showing only related list Ids for that customer?

**Comment by Innovative-GauravKochar** [comment]
@AnkitSegment Can you please test and share e2e flow of your changes like showing some evidences of using existing audience and syncing them successfully ?

File: packages/destination-actions/src/destinations/klaviyo/generated-types.ts (hunk 1 above)
Comment (AnkitSegment) [reply packages/destination-actions/src/destinations/klaviyo/generated-types.ts:16]: Audience fields do not have options to add dynamic fields.

**Comment by joe-ayoub-segment** [comment]
hi @AnkitSegment PR deployed

**Pull request merged**